saos-webapp/src/main/resources/message/common.properties

```diff
@@ -76,7 +76,7 @@ sort.pageSize = Pozycji na stronie
 sort.by = Sortuj po
 sort.date = Data
 sort.accuracy = Trafność
-sort.referencingCount = Liczba odwołań
+sort.referencingCount = Przywołania sprawy
```

Comment:
I think 'Liczba przywołań' would be better (or even 'Liczba przywołań sprawy' but this could be too long)? I would ask Olek what he thinks about it.